src/SQLite.jl

```diff
@@ -286,14 +373,16 @@ function sqlitevalue(::Type{T}, handle, col) where {T}
     return r
 end
 
-sqlitetype(::Type{T}) where {T<:Integer} = "INT NOT NULL"
-sqlitetype(::Type{T}) where {T<:Union{Missing, Integer}} = "INT"
-sqlitetype(::Type{T}) where {T<:AbstractFloat} = "REAL NOT NULL"
-sqlitetype(::Type{T}) where {T<:Union{Missing, AbstractFloat}} = "REAL"
-sqlitetype(::Type{T}) where {T<:AbstractString} = "TEXT NOT NULL"
-sqlitetype(::Type{T}) where {T<:Union{Missing, AbstractString}} = "TEXT"
+# conversion from Julia to SQLite3 types
+sqlitetype_(::Type{<:Integer}) = "INT"
```

Comment:
I would very much prefer to avoid  introducing `sqlitetype_()` method too. The problem is, `sqlitetype()` has to return *"T NOT NULL"* for non-missing types and *"T"* for missing ones. Which means the pattern `sqlitetype(::Type{Union{T, Missing}) = f(sqlitetype(T))` will not work here. Of all solutions to this problem the one I implemented looked like the most straightforward one. But I can change it if you have better ideas.